frontend/src/components/auth/Login.tsx

```diff
@@ -48,6 +48,20 @@ const Login = () => {
     setSignup(true);
   };
 
+  type GoogleErrorResponse = {
+    error: string;
+    details: string;
+  };
+
+  const onFailure = (response: GoogleErrorResponse) => {
+    // https://stackoverflow.com/questions/63631849/google-sign-in-not-working-in-incognito-mode
+    if (response.error === "idpiframe_initialization_failed") {
+      console.warn("Google SignIn does not work on incognito mode.");
+    } else {
+      alert(JSON.stringify(response));
```

Comment:
just alerting an object leads to `[object Object]` which gives the user no information. `JSON.stringify(response)` will actually show the full response (eg `{"error":"idpiframe_initialization_failed","details":"Cookies are not enabled in current environment."}`